glossary.md

```diff
@@ -6,28 +6,97 @@
+* `Description`: Short summary of the object, including geographic or thematic scope. This could also be 
+  referred to as an abstract (see Arctic Data Center). 
+* `Contact Name`: The name of a point of contact for the object, e.g. a data manager or program coordinator.
+* `Contact Title`: The title of a point of contact for the object.
+* `Contact Email`: The email address for a point of contact for the object.
+* `Tags`: This is a user-defined field that admins can and will edit. It creates additional flexibility in 
+  the analyis, for example allowing for a regionally- or thematically-focused `analysis`.
+* `Version`: This field allows the `analyses` to reflect updates overtime. Likely
+  this field will have to be open text so it can match the versioning information that the organization uses.
+* `Persistent Identifier`: A standard way to refer back to the object's source, usually a ROR or DOI
+
+`Application` objects also include:
+* `Application Performance Criteria`: Text description of what the ideal performance of this data
+  product looks like.
+
+## Rated Instance of an Object
+
+* _NOTEfromHAZEL_ I think we can consolidate many of the sections below into this single
+  description of links between the `Observing System`, `Data Product`, and `Application` 
+  objects. Links to `Societal Benefit Areas` require separate definitions.
+
+This answers the questions - how important is a particular `Observing System`, 
+`Data Product`, or `Application`and how well does it perform. It is reflected in the 
+`analysis`by the thicknessand color of the link connecting two `survey objects`. For 
+instance: Imagine a satelite (`observing system`) that is very important to a sea ice 
+`data product` but performs poorly because of persistent Arctic cloud cover and high 
+latency (`gaps`. Those would be linked by a thick (high `criticality`) red (low 
+`performance`) line. 
+
+Response objects exist both as a definition in the `registry` and an instantiation with
+rating(s) and other fields associated with a `response`. The following specifications
+pertain to rated instances, _not_ `registry` definitions.
+
+* `Link`: Defines which `survey objects` are connected. The rating is applied to that `link`.
+  _TODO_: Research ontologies around links/provinence - 
+  https://www.w3.org/TR/prov-o/#cross-reference-starting-point-terms
+* `Performance rating`: 0-100 rating of the performance of the subject. Answers the question: 
+  What is your satisfaction with this input? (0=No performance, 100 = perfect)
+* `Criticality rating`: 0-10 rating of the criticality of an input to an output,
+  e.g. criticality of an `observing system` to a `data product`. This answers the question:
+  On a scale of 1-10, how much would the loss of this input impact the performance of your 
+  `data product` or `application` (1 - very little impact; 10 - complete loss of performance).
+* `Rationale`: Why a `criticality` or `performance` rating was selected.
```

Comment:
Should there be two rationale fields, one for criticality and one for performance?

Reply:
Yes, we've only had one field in the past but I think as we move towards a more standard structure that makes sense

Reply:
- [ ] Create two `rationale` fields